easybuild/easyconfigs/h/HOLE2/HOLE2-2.3.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'HOLE2'
+version = '2.3.1'
+
+homepage = 'https://www.holeprogram.org/'
+description = """HOLE is a program that allows the analysis and visualisation of
+the pore dimensions of the holes through molecular structures of ion channels"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
```

Comment:
Hm, i wonder if GCC is a better level for this (and if so, remember to drop the binutils builddep, which is included with GCC already)

Reply:
I thought this but I am not sure whether I should use GCC to have binutils. zlib is not used.